.github/workflows/application-onboarding-or-promotion-automation.yml

```diff
@@ -0,0 +1,95 @@
+name: Automation for application onboarding or promotion
+
+on:
+  workflow_dispatch:
+    inputs:
+      application_name:
+        description: Application name
+        required: true
+        type: string
+      application_environment:
+        description: Application environment
+        required: true
+        type: string
```

Comment:
Should we make this a dropdown with a list of `options`?

Reply:
This is an action, not the actual form. The form does have a dropdown with options.

![image](https://user-images.githubusercontent.com/56260532/185437368-2d7cf1f2-19d3-4492-9e75-44a0bb59568a.png)

The same will be true for the onboarding form once these changes are merged, although it only has one option of `dev`

![image](https://user-images.githubusercontent.com/56260532/185437742-d3510531-807e-44c1-b5ba-4fe291a23622.png)

Reply:
I was thinking a dropdown could be in the `workflow_dispatch`:

![image](https://user-images.githubusercontent.com/10993987/185475864-1a49de54-add1-462e-a4e8-c7e558880724.png)

Reply:
Good call, will look into it!